gui/tabs/players_tab.cpp

```diff
@@ -84,6 +84,13 @@ namespace PlayersTab {
 					}
 
 					if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer) {
+						if (!GetPlayerData(State.selectedPlayer)->fields.IsDead && GetPlayerData(*Game::pLocalPlayer)->fields.IsImpostor 
+							&& !GetPlayerData(*Game::pLocalPlayer)->fields.IsDead && !State.InMeeting) //Found game will black screen if murdering in a meeting
```

Comment:
I'd highly advise adding a kill cooldown check.

Reply:
Understandable, but I've murdered the entire game with no bans yet.